sklearn/metrics/cluster/_unsupervised.py

```diff
@@ -109,6 +109,34 @@ def silhouette_score(
     silhouette : float
         Mean Silhouette Coefficient for all samples.
 
+    Examples
+    --------
+    >>> from sklearn.datasets import make_blobs
```

Comment:
Let's simplify the example. We should only call `silhouette_score` once and show the floating output.

The user guide and the example gallery is dedicated to make complex pattern.

Reply:
How do I make the changes to this pull request. I am learning. I would really appreciate if you can guide me

Reply:
- pull from your remote
- modify the file locally accordingly to the suggestion
- make a local commit
- push your branch into your remote

However, I see that you modified your `main` branch. This is something that you should not do. You should always a pull request from a different branch from the default one.

You can have a look at this repository that we use when teaching how to contribute to open-source. Here, we are dealing with the step 5. and above.

Reply:
Are you talking about this 'https://scikit-learn.org/dev/developers/contributing.html'?

Reply:
Ups, I forgot the link: https://github.com/scikit-learn-inria-fondation/EuroSciPy22

This is similar explanation but more gradual.

Reply:
I have updated the example. Should I create a new pull request giving this as a reference?

Reply:
Normally you should push into this remote branch. In case you created a new branch, feel free to close this PR and open a new one.

Reply:
Sorry just getting started. I will keep it in mind in the future. Thanks for the help